[2025-09-06T05:31:21.619+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:31:18.683434+00:00 [queued]>
[2025-09-06T05:31:21.646+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:31:18.683434+00:00 [queued]>
[2025-09-06T05:31:21.646+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-09-06T05:31:21.708+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): validate_model> on 2025-09-06 05:31:18.683434+00:00
[2025-09-06T05:31:21.733+0000] {standard_task_runner.py:57} INFO - Started process 1685 to run task
[2025-09-06T05:31:21.761+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mysql_insert_select', 'validate_model', 'manual__2025-09-06T05:31:18.683434+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/mysql_insert_select.py', '--cfg-path', '/tmp/tmpjogvo6o6']
[2025-09-06T05:31:21.765+0000] {standard_task_runner.py:85} INFO - Job 292: Subtask validate_model
[2025-09-06T05:31:21.922+0000] {task_command.py:410} INFO - Running <TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T05:31:18.683434+00:00 [running]> on host 6faa580ba6cb
[2025-09-06T05:31:22.203+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_insert_select' AIRFLOW_CTX_TASK_ID='validate_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T05:31:18.683434+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T05:31:18.683434+00:00'
[2025-09-06T05:31:22.205+0000] {logging_mixin.py:149} INFO - ✅ Modelo cargado correctamente
[2025-09-06T05:31:22.206+0000] {logging_mixin.py:149} INFO -    - Tipo: <class 'numpy.ndarray'>
[2025-09-06T05:31:22.206+0000] {logging_mixin.py:149} INFO -    - Características esperadas: No disponible
[2025-09-06T05:31:22.206+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-09-06T05:31:22.229+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=mysql_insert_select, task_id=validate_model, execution_date=20250906T053118, start_date=20250906T053121, end_date=20250906T053122
[2025-09-06T05:31:22.296+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-09-06T05:31:22.339+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
